try:
     #print(mi_variable)
     10/0
except Exception as e:
     print("ocurrio un error: {e}")
